src/main/java/org/folio/search/configuration/JsonPathConfiguration.java

```diff
@@ -0,0 +1,63 @@
+package org.folio.search.configuration;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.Option;
+import com.jayway.jsonpath.ParseContext;
+import com.jayway.jsonpath.spi.json.JacksonJsonProvider;
+import com.jayway.jsonpath.spi.json.JsonProvider;
+import com.jayway.jsonpath.spi.mapper.JacksonMappingProvider;
+import com.jayway.jsonpath.spi.mapper.MappingProvider;
+import java.util.EnumSet;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
```

Comment:
@pfilippovEPAM, @MikhailFokanov, has it been agreed to add java docs for all the classes and public methods? As far as I know it was not required on FOLIO before.

Reply:
It is not mandatory in Folio. I would say developer need to add Javadoc, where there is some sophisticated business logic or classes. Just not to add javadoc everywhere just for javadoc.

Reply:
Yeah, I agree.